packages/common/driver-definitions/api-report/driver-definitions.api.md

```diff
@@ -238,6 +238,20 @@ export interface ILocationRedirectionError extends IDriverErrorBase {
     readonly redirectUrl: IResolvedUrl;
 }
 
+// @alpha
+export interface IPartialSnapshotWithContents {
+    // (undocumented)
+    blobsContents: Map<string, ArrayBuffer>;
+    // (undocumented)
+    couldBePartialSnapshot: true;
```

Comment:
Why "could"? "is" a better verb here. Though I'd not use verbs here, just partialSnapshot = true.

Reply:
renamed